conf/boards/apogee_1.0.makefile

```diff
@@ -36,6 +36,7 @@ SYS_TIME_LED       ?= 1
 
 MODEM_PORT ?= UART1
 MODEM_BAUD ?= B57600
+MODEM_CHANNEL ?= 0
```

Comment:
This is not board specific, but drone specific. A bit strange to have this configure here.